R/ggpairs_add.R

```diff
@@ -13,15 +13,15 @@
 ######   \item \code{uneval}: replace current aesthetics
 ######   \item \code{layer}: add new layer
 #'   \item \code{theme}: update plot theme
-######   \item \code{scale}: replace current scale
-######   \item \code{coord}: override current coordinate system
+#'   \item \code{scale}: replace current scale
+#'   \item \code{coord}: override current coordinate system
```

Comment:
These two items are not true for `+.gg`. (Only for `add_ggproto_to_ggmatrix`.). 

It will cause issues when someone doesn't supply the correct locations.  

Maybe it is worth splitting the two documentations? They can link to each other with a `@seealso \code{\link{FUNC}}`

Reply:
** Correction.

When I mean cause issues, I've seen users try to use `scale_x_log10` when they have some columns containing discrete data.  This does make sense for the context of colors/fills.  

Is there a way to check for this?

Reply:
I'm not sure to have fully understood your two comments.

> These two items are not true for `+.gg`. (Only for `add_ggproto_to_ggmatrix`.).

`+.gg()` have been modified to call `add_ggproto_to_ggmatrix()`




> When I mean cause issues, I've seen users try to use `scale_x_log10` when they have some columns containing discrete data.

But there are other cases where it's appropriate to do it, for example, if you have only continuous variables. 

If you do `ggpairs(tips) + scale_x_log10()` you simply have an error.  As you would have with `ggplot(tips) + aes(x = sex) + geom_bar() + scale_x_log()`.

And if you want to modify just certain type of subplots, then you have the option of using specifically `add_ggproto_to_ggmatrix()` with the argument `only_fn`.

Once again, good documentation is probably key and a vignette around `ggpairs()` a good thing to develop once documentation moved to `pkgdown`.

Regards

Reply:
Ok.  Might come back to this later if we run into more problems. 

But it does open a lot of opportunity.